add-on/chrome/bootstrap.js

```diff
@@ -299,7 +299,9 @@ var WindowListener = {
           }
         });
 
-        this.addMenuItem();
+        if (!PrivateBrowsingUtils.isWindowPrivate(window)) {
+          this.addMenuItem();
+        }
```

Comment:
Shouldn't we add a return statement here? If we're in a private window and the icon is hide, we shouldn't add observers and the other stuff, right?